balances-adjustment/src/test/java/com/powsybl/balances_adjustment/balance_computation/BalanceComputationSimpleDcTest.java

```diff
@@ -225,9 +225,11 @@ void testSkipLoadFlowSingleIteration() {
         // The result should be SUCCESS since we didn't run load flow to verify convergence
         assertEquals(BalanceComputationResult.Status.SUCCESS, result.getStatus());
         // Should have done exactly 1 iteration
```

Comment:
Remove this comment or edit it
```suggestion
        // Should have done exactly 2 iterations
```

Reply:
done